addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -41,6 +41,8 @@ odoo.define('website.s_website_form', function (require) {
         xmlDependencies: ['/website/static/src/xml/website_form.xml'],
         events: {
             'click .s_website_form_send, .o_website_form_send': 'send', // !compatibility
+            'change input[type=file]': '_onFileChange',
+            'click input[type=button]': '_onfakeButtonClick',
```

Comment:
`_onfakeButtonClick` => `_onFakeButtonClick`
Wouldn't this handler also be called when the submit button is clicked ? Maybe the selector should be more specific.